windows-find-tools/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "windows-find-tools"
```

Comment:
A few alternative naming ideas:
- `windows-tools`
- `find-msvc-tools`
- `msvc-tools`

I'd defer to @ChrisDenton for which name actually makes the most sense.